### ISSUE TITLE
Support for Vaadin 7

### ISSUE THREAD

**Issue by jensim** [jouni/Snappy#1]
A lot of things changed in Vaadin 7, and development will be needed

**Comment by jouni** [comment]
Correct. A quick approach would be to just convert Snappy to implement LegacyComponent and LegacyConnector. That should be at least the starting point, but I haven't found the time and inspiration to do even that yet.

**Comment by jensim** [comment]
I made a fork of the repo, and i'll see if i've got what it takes to get it up and running. I really love the features it provides!

**Comment by jouni** [comment]
Fear not, there is still hope for a Vaadin 7 version! The next version (2.0) has been cooking up in the [Animator](https://github.com/jounik/Animator) repo, but I'll split it up before the release.

So Snappy 2.0 require the Animator 2.0 add-on as well, but you will want to use that once you see what the two can do together ;)
